### PR TITLE
Update 'with_salary' scope

### PR DESCRIPTION
**Pull request by TobyRet**
### Context
_Apply_ trello card: https://trello.com/c/LvY0EtlQ/2332-bug-salaried-courses-filter 
Slack thread: https://ukgovernmentdfe.slack.com/archives/CQA64BETU/p1602522466230000

London Metropolitan University is an accredited body for some courses. When a user searches by provider ('London Metropolitan University') and then filters the results by 'courses with salary' then any courses that are apprenticeships are not being returned.

Find considers salaried courses to include apprenticeships.
https://github.com/DFE-Digital/find-teacher-training/blob/master/app/decorators/course_decorator.rb#L78

### Changes proposed in this pull request
Return apprenticeship courses when calling the `with_salary` scope on the Course model

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.7.2
```
Comment (dankmitchell) [inline .ruby-version:1]: Controversial

**Review by dankmitchell — APPROVED** [review]
LGTM but you'll need to remove the `.ruby-version` file

**Review by defong — APPROVED** [review]

**Review by tvararu — APPROVED** [review]

File: .ruby-version (hunk 1 above)
Comment (TobyRet) [reply .ruby-version:1]: Woops - must have slipped in there after I upgraded
https://github.com/DFE-Digital/teacher-training-api/commit/ecb382333e3d38b75ccca4377b3fe035e4c26f9f

**Comment by TobyRet** [comment]
> LGTM but you'll need to remove the `.ruby-version` file

@dankmitchell - Removed and updated `.gitignore`
https://github.com/DFE-Digital/teacher-training-api/pull/1605/commits/4de6144ba3fcd4431c4d881c06cbec291b3030fe

**Review by dankmitchell — APPROVED** [review]

**Pull request merged**